features/src/python-lit/install.sh

```diff
@@ -26,7 +26,10 @@ check_packages                  \
     software-properties-common  \
     ;
 
-apt-get install -y --no-install-recommends python3 python3-dev python3-wheel python3-pip;
+if ! type python >/dev/null 2>&1; then
+  echo "lit feature expects python to already be installed" >&2;
+  exit 1;
+fi
 python3 -m pip install wheel;
```

Comment:
pip errors from some kind of dependency thing if you try to install `wheel` and `lit` at the same time. This is only necessary on 18.04 and maybe 20.04?

`psutil` is needed for `lit` to kill tests that are timing out.

Reply:
> pip errors from some kind of dependency thing if you try to install `wheel` and `lit` at the same time

Curious why we need `wheel`? Is lit compiled from an sdist, or does it implicitly depend on `wheel` but omit it from its build requirements?